### ISSUE TITLE
There seems to be an issue with the new provisioning on develop branch.

### ISSUE THREAD

**Issue by kopepasah** [Varying-Vagrant-Vagrants/VVV#1045]
Recent updates to the `develop` branch are not provisioning any custom sites (currently debugging). When switching to the latest tag (currently `1.4.0`) the provisioning works correctly, so my thoughts are this is caused by a recent merge. I did notice there were many updates made to the `provision/provision.sh` in a recent merge, so my discovery process will start in that location.

If no one else is experiencing this issue, please let us know.

**Comment by kopepasah** [comment]
@LoreleiAurora I can see now that the new provisioning system currently provisions sites from the `vvv-config.yml` and I am curious of two things:

1. Will sites with `vvv-init.sh` files no longer provision automatically?
2. Is there a specific reason we changed the `-maxdepth` from `5` to `4`?

**Comment by LoreleiAurora** [comment]
@kopepasah The docs for the new provisioning system are currently being written.

All sites you wish to provision must have there __directory name__ added to `vvv-config.yml`.
For example a site in a directory of `www/foo-site` most have at lease `foo-site:` added to vvv-config.yml.

All options available to a site can currently be viewed in #980.

**Comment by kopepasah** [comment]
@LoreleiAurora yes, I saw that information. What I am currently not seeing is regarding number four of @jeremyfelt's comment [here](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/980#issuecomment-258346793). We should not require users to edit a file (i.e. `vvv-config.yml`) which is not ignored by the repo.

While I do agree with Jeremy's suggestion to also include a `vvv-custom.yml`, I would go a step further and say only load the `vvv-custom.yml` if it exists. This way, a user can exclude WordPress default and develop sites, if she/he so chooses.

I have created a PR which addresses this issue: https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1046
